website/pages/docs/configuration/storage/aerospike.mdx

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
It also needs an entry in [this file](https://github.com/hashicorp/vault/blob/master/website/data/docs-navigation.js#L67) so that it can be included in the sidebar.

Reply:
Looks like Jim was referring to the same thing in the comment above, resolving this one!